src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -479,7 +482,7 @@ private static LocalDate deriveOffenderReleaseDate(final KeyDateValues keyDateVa
 
 
     public Optional<LocalDate> getConfirmedReleaseDate() {
-        return Optional.ofNullable(releaseDetail != null ? releaseDetail.getReleaseDate() != null ? releaseDetail.getReleaseDate() : releaseDetail.getAutoReleaseDate() : null);
+        return Optional.ofNullable(releaseDetail != null ? releaseDetail.getReleaseDate() : null);
```

Comment:
The old SQL solution didn't default auto release date into confirmed release date if release date missing